src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+:root {
+  --colorBlye: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.header {
+  max-width: 100%;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  display: flex;
+  margin: auto;
+  position: relative;
+}
+
+.logo {
+  position: absolute;
+  left: 50px;
+  top: 10px;
+}
+
+.nav__item:not(:first-child) {
+  margin: 0 0 0 20px;
+}
+
+/*.is-active {
+  position: relative;
+  color: #00acdc;
+  margin: 0;
```

Comment:
Please uncomment this part of code, active text color should be blue 
![image](https://user-images.githubusercontent.com/54936044/236057119-99fba35b-299c-48b7-a1cc-b69b54258bb4.png)